amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -793,11 +781,6 @@ describe('SearchPage', () => {
       });
     });
 
```

Comment:
Add a test for `SearchPanel` exists and `renderContent()` is called? Also fine if you put a `TODO` in the component file to cover these cases.